src/java.sql/share/classes/java/sql/Connection.java

```diff
@@ -1603,8 +1593,8 @@ default void endRequest() throws SQLException {
      * @see ShardingKeyBuilder
      */
     default boolean setShardingKeyIfValid(ShardingKey shardingKey,
-            ShardingKey superShardingKey, int timeout)
-            throws SQLException {
+                                          ShardingKey superShardingKey,
+                                          int timeout) throws SQLException {
```

Comment:
I haven't quite identified your preferred formatting style.
From previous reformattings with multiple arguments, the exception was placed on a line by itself.

(Generally, we defer to the original/previous author's choice of line breaks and formatting. 
Choosing only to reformat to use a consistent style within each file/class or package.)

The issue title and PR title should be a bit more general, its more than just indentation cleanup.

Reply:
@RogerRiggs My preferred formatting style is like this: 

```java
    default boolean setShardingKeyIfValid(ShardingKey shardingKey,
                                          ShardingKey superShardingKey,
                                          int timeout) throws SQLException {
```

But as you said, there is previous reformattings that does not follow it. If you agree, I will fix them. What do you thing about it?

About the issue title and PR title, we can make it more general like: 

"Improve Readability and Maintainability of Connection.java"

What your suggestion on it?

Reply:
Great! I will do that.